prep.sh

```diff
@@ -41,7 +41,7 @@ for LINE in $LINEZ; do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=1152.vafc19b26d5aa # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/345
+version=1175.vb05220ee3c55 # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/375
```

Comment:
```suggestion
version=1178.vbef3c43d0e69
```

Reply:
1 second apart 🤣

Reply:
Yeah, hopefully this is the latest round of testing I need to do on this PR. Now comes the hard part: writing the PR description for my non-draft PR. I always think that writing the code is easy compared to writing the PR description.

Reply:
@basil PR description suggestion: `Make shit work with proper testing and more correct hackz`